test/attr/attr_objc.swift

```diff
@@ -1264,16 +1264,16 @@ class infer_instanceVar1 {
   var var_ExistentialMetatype7: (Protocol_Class1 & Protocol_ObjC1).Type
   var var_ExistentialMetatype8: Protocol_ObjC1.Type
   var var_ExistentialMetatype9: (Protocol_ObjC1 & Protocol_ObjC2).Type
-// CHECK-LABEL: {{^}}  var var_ExistentialMetatype0: Any.Type
-// CHECK-LABEL: {{^}}  var var_ExistentialMetatype1: PlainProtocol.Type
-// CHECK-LABEL: {{^}}  var var_ExistentialMetatype2: (PlainProtocol).Type
-// CHECK-LABEL: {{^}}  var var_ExistentialMetatype3: (PlainProtocol & Protocol_Class1).Type
-// CHECK-LABEL: {{^}}  var var_ExistentialMetatype4: (PlainProtocol & Protocol_ObjC1).Type
-// CHECK-LABEL: {{^}}  var var_ExistentialMetatype5: (Protocol_Class1).Type
-// CHECK-LABEL: {{^}}  var var_ExistentialMetatype6: (Protocol_Class1 & Protocol_Class2).Type
-// CHECK-LABEL: {{^}}  var var_ExistentialMetatype7: (Protocol_Class1 & Protocol_ObjC1).Type
-// CHECK-LABEL: @objc var var_ExistentialMetatype8: Protocol_ObjC1.Type
-// CHECK-LABEL: @objc var var_ExistentialMetatype9: (Protocol_ObjC1 & Protocol_ObjC2).Type
+// CHECK-LABEL: {{^}}  var var_ExistentialMetatype0: any Any.Type
+// CHECK-LABEL: {{^}}  var var_ExistentialMetatype1: any PlainProtocol.Type
```

Comment:
Just double-checking: Is `any PlainProtocol.Type` correct here or should this be `(any PlainProtocol).Type`? I don’t know what the right answer is.

Reply:
The existential metatype is `any PlainProtocol.Type`, yes.

Reply:
OK. Thanks for the clarification. 👍🏽